### PR TITLE
Card two and five

### PR DESCRIPTION
**Pull request by Gmfholley**
Adds cards 2 and 5.

Also:
* moves Card 1 to the design page

Nonhover state:
Screen Shot 2021-11-14 at 9 30 21 AM

Hover state:
Screen Shot 2021-11-14 at 9 31 54 AM

### CODE REVIEW

File: assets/styles/1-design-tokens/_variables--color.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* Create the CSS Variables for White, Black, and Grays */
 :root {
-    --color-white:         255,255,255;//#ffffff    // black ratio 17.92    // white ratio ----
+    --color-white:         rgb(255,255,255);//#ffffff    // black ratio 17.92    // white ratio ----
```
Comment (Gmfholley) [inline assets/styles/1-design-tokens/_variables--color.scss]: The buttons PR does this for all colors as well.

File: assets/styles/1-design-tokens/_variables--spacing.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* Create the Global Spacing CSS Variables */
 // Base = 8px (Spacing for mobile)
 :root{
-    --space-unit: 0.5;
+    --space-unit: 1.0;
```
Comment (Gmfholley) [inline assets/styles/1-design-tokens/_variables--spacing.scss]: So I found that none of the spacing was accurately described unless the space unit was 1.0.  So I corrected it.

If I'm missing something here, please correct me.  But this allowed me to trust the comments on the side about how large/small things were.

Comment (tgawley) [reply assets/styles/1-design-tokens/_variables--spacing.scss]: @Gmfholley I *think* it was set up for base 8px or .5 on mobile and then 16px or 1 on desktop, but if that's not what's happening could you show me for my own knowledge and learning. Thanks!

Comment (Gmfholley) [reply assets/styles/1-design-tokens/_variables--spacing.scss]:
```suggestion
    --space-unit: 0.5;
```

File: assets/styles/6-components/_card.scss [hunk 3]
```diff
@@ -22,4 +22,78 @@
...
+
+        .card__title {
+            text-align: left;
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            padding: var(--space-inset-medium);
+            margin: 0;
+            font-family: var(--font-family-share-tech-mono);
+            color: var(--color-white);
```
Comment (Gmfholley) [inline assets/styles/6-components/_card.scss:93]: Is it okay that I am missing padding and margin in these ways?

**Review by mel-d — APPROVED** [review]

**Pull request merged**